docs/src/catalyst_functionality/dsl_description.md

```diff
@@ -412,6 +412,74 @@ sol = solve(oprob)
 plot(sol)
 ```
 
+## Constant/fixed species
+It is possible to fix the concentration of a species in a reaction. Without fixing
```

Comment:
```suggestion
It is possible to fix the amount of a species in a reaction. Without fixing
```
Since we support either concentration or "number of" units.